lib/posts.js

```diff
@@ -220,4 +221,48 @@ PostsRequest.prototype.statuses = function() {
 	return this;
 };
 
+/**
+ * @method year
+ * @chainable
+ * @param year - Integer for year
+ * @returns {CollectionRequest} The PostsRequest instance (for chaining)
+ */
+PostsRequest.prototype.year = function ( year ) {
+  return this.filter( 'year', year );
+};
+
+/**
+ * @method month
+ * @chainable
+ * @param month - Integer for month or month string ("january")
+ * @returns {CollectionRequest} The PostsRequest instance (for chaining)
+ */
+PostsRequest.prototype.month = function ( month ) {
+  if ( _.isString( month ) ) {
+    month = new Date( Date.parse(month +" 1, 2012") );
+
+    if ( isNaN(month) ) {
+      return this;
+    }
+
+    month = month.getMonth() + 1;
+  }
+
+  if ( _.isNumber( month ) ) {
+    return this.filter( 'monthnum', month );
+  }
+
+  return this;
+};
+
+/**
```

Comment:
We've been trying to leave brief, human-readable descriptions of API methods as the first line of a docblock, to aid in auto-generated documentation.